main.nf

```diff
@@ -325,23 +326,23 @@ if( workflow.profile == 'awsbatch') {
 }
 
 // Stage config files
-ch_multiqc_config = Channel.fromPath(params.multiqc_config, checkIfExists: true)
-ch_output_docs = Channel.fromPath("$baseDir/docs/output.md", checkIfExists: true)
+ch_multiqc_config = file(params.multiqc_config, checkIfExists: true)
```

Comment:
I think this is behaving badly. Or maybe the `.collect()` that ingests it.

Reply:
Its definitely the `collect()` because I have been using it this way for a while now.